xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/job/MoveRequest.java

```diff
@@ -19,27 +19,20 @@
  */
 package org.xwiki.refactoring.job;
 
-import org.xwiki.model.reference.EntityReference;
-
 /**
  * A job request that can be used to move a collection of entities to a specified destination. This request can also be
  * used to rename an entity.
  * 
  * @version $Id$
  * @since 7.2M1
  */
-public class MoveRequest extends EntityRequest
+public class MoveRequest extends AbstractCopyOrMoveRequest
```

Comment:
Seems to me deleteSource related APIs should be deprecated since it does not make much sense for a move now.

Reply:
Rename will delete source

Reply:
Yes but point was more that there is no real use case where we don't want to remove the source anymore.

Reply:
Yeah you're right.